cfme/cloud/instance.py

```diff
@@ -347,12 +340,6 @@ def create(self, email=None, first_name=None, last_name=None, availability_zone=
             flash.assert_success_message(
                 "VM Provision Request was Submitted, you will be notified when your VMs are ready")
 
-        row_description = 'Provision from [{}] to [{}]'.format(self.template_name, self.name)
```

Comment:
Why remove these asserts? They were there to ensure that the task showed the right output status. Is there an issue with them?